packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -74,46 +73,55 @@ contract L2Genesis is Script, Artifacts {
         0xa0Ee7A142d267C1f36714E4a8F75612F20a79720
     ];
 
-    string internal outfile;
-
-    /// @dev Reads the deploy config, sets `outfile` which is where the `vm.dumpState` will be saved to, and
-    ///      loads in the addresses for the L1 contract deployments.
-    function setUp() public override {
-        Artifacts.setUp();
-
-        string memory path = string.concat(vm.projectRoot(), "/deploy-config/", deploymentContext, ".json");
-        vm.etch(address(cfg), vm.getDeployedCode("DeployConfig.s.sol:DeployConfig"));
-        vm.label(address(cfg), "DeployConfig");
-        vm.allowCheatcodes(address(cfg));
-        cfg.read(path);
-
-        outfile = string.concat(vm.projectRoot(), "/deployments/", deploymentContext, "/genesis-l2.json");
+    function name() public pure override returns (string memory) {
+        return "L2Genesis";
+    }
 
-        _loadAddresses(string.concat(vm.projectRoot(), "/deployments/", deploymentContext, "/.deploy"));
+    function artifactDependencies() internal view returns (L1Dependencies memory l1Dependencies_) {
+        console.log("retrieving L1 deployments from artifacts");
+        return L1Dependencies({
+            deployedL1CrossDomainMessengerProxy: mustGetAddress("L1CrossDomainMessengerProxy"),
+            deployedL1StandardBridgeProxy: mustGetAddress("L1StandardBridgeProxy"),
+            deployedL1ERC721BridgeProxy: mustGetAddress("L1ERC721BridgeProxy")
```

Comment:
smol nit: i don't think we need the `deployed` prefix, could just add a comment on the function name